TheEthicalHackerCup/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2021.3.12f1
-m_EditorVersionWithRevision: 2021.3.12f1 (8af3c3e441b1)
+m_EditorVersion: 2021.3.16f1
+m_EditorVersionWithRevision: 2021.3.16f1 (4016570cf34f)
```

Comment:
Did you update Unity? This might cause some complications for the rest of us who are still on 2021.3.12f1

Reply:
Version reverted